nixos/modules/services/matrix/synapse/synapse.md

```diff
@@ -113,6 +113,31 @@ in {
 }
 ```
 
+### Workers {#module-services-matrix-synapse-workers}
+
+The synapse module also supports instantiating workers via the `services.matrix-synapse.workers.instances` option.
```

Comment:
```suggestion
The synapse module also supports instantiating workers via the `services.matrix-synapse.workers` option.
```
if we remove the options prefixed with main we can also remove the duplication in the name here

Reply:
I guess since there's no longer any global worker options to set there's an argument to have the `federationSenders = n` just in the root module rather than in a `workers` submodule

It might be harder to split the file like this though, since the workers.nix file being the whole `workers` attribute is very simple currently.

Would prefer input from more maintainers on this.